Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:7.4.10-cli
+FROM php:cli
```

Comment:
Being a development tool; it's probably ok to just track latest stable and automatically yield any performance improvements etc from upcoming versions. If you start to experience issues due to BC breaks and want to control this more closely you could `FROM php:7-cli` (allowing new features, bugfixes but not BC Breaks until you're ready)